packages/webapp/src/components/List/index.jsx

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
`List` sounds a bit generic to me in the sense that I could imagine us having other kinds of list components in the future. Should we maybe call this something like `TileList`, or similar?

Reply:
But wouldn't other list components be described as a different listItems? List is meant to be very generic in the sense it is just a row that takes up 100% width --there is almost no styling on this component. A tile would be added to the tile types and have a matrix formation.

Reply:
Ah yeah, that's a good point! I'm on board 🙂